wmf2svg/src/net/arnx/wmf2svg/Main.java

```diff
@@ -34,29 +34,50 @@
 	private static Logger log = Logger.getLogger(Main.class.getName());
 
 	public static void main(String[] args) {
+		FontFactory factory = FontFactory.getInstance();
+		factory.fingerprintFont();
 		String src = null;
 		String dest = null;
-
+		String fontsRegEntryPath = null;
 		boolean debug = false;
 		boolean compatible = false;
 		boolean replaceSymbolFont = false;
 
 		for (int i = 0; i < args.length; i++) {
-			if (args[i].startsWith("-")) {
-				if (args[i].equals("-debug")) {
+			if (args[i].startsWith("-"))
+			{
+				if (args[i].equals("-debug"))
+				{
 					debug = true;
```

Comment:
This break line style is inappropriate. A line break shouldn't be placed before '{'  in a common java coding style.

Reply:
Sorry. I am correcting it and will update in 2nd code pull request